package.json

```diff
@@ -74,5 +65,56 @@
 		"webpack-cli": "^2.1.2",
 		"webpack-dev-server": "^2.9.1",
 		"webpack-merge": "^4.1.2"
-	}
+	},
+	"eslintConfig": {
+		"extends": [
+			"plugin:vue/essential",
+			"@vue/prettier"
+		],
+		"root": true,
+		"rules": {
+			"no-debugger": 0
+		}
+	},
+	"license": "MIT",
+	"lint-staged": {
+		"*.{js,json,css,scss,md,vue}": [
+			"prettier --write",
+			"git add"
+		]
+	},
+	"name": "@3yourmind/kotti",
+	"prettier": {
```

Comment:
Did not we decide company-wide for this configuration:

```json
{
  "useTabs": true,
  "singleQuote": true,
  "trailingComma": "all",
  "arrowParens": "always"
}
```

We should be consistent in our style in all our code bases in my opinion.

Reply:
@igeligel I copied most of it from the old `.prettierrc`. I'm fine with using `"arrowParens": "always"` though.

Reply:
Yeah, the old prettier configuration file was not up to date. Like I said, we should use the configuration which I posted here in all our projects.